kaggle_environments/main.py

```diff
@@ -63,6 +63,9 @@
 parser.add_argument(
     "--host", type=str, help="http-server Host (default=127.0.0.1)."
 )
+parser.add_argument(
+    "--out", type=str, help="Output file to write the results of the episode."
```

Comment:
This allows you to write the results to an html file while still getting print statements in std out.